puppet/modules/slave/manifests/init.pp

```diff
@@ -229,10 +233,9 @@
   include slave::java
 
   # Databases
-  include slave::mysql, slave::postgresql
-  slave::db_config { 'mysql': }
+  include slave::mysql
```

Comment:
This is still used by something?

Reply:
If you notice the actual class you notice that it now ensures the package and db config are absent. That means after this has been rolled out we can remove it after a while. I assumed that a package removal stops the service. It doesn't remove the actual storage (DBs etc).

Reply:
add a "#TODO remove when all nodes have been cleaned"?